src/paste3/helper.py

```diff
@@ -214,3 +233,29 @@ def kl_divergence_backend(X, Y):
     X_log_X = nx.reshape(X_log_X, (1, X_log_X.shape[0]))
     D = X_log_X.T - nx.dot(X, log_Y.T)
     return nx.to_numpy(D)
+
+
+def dissimilarity_metric(which, sliceA, sliceB, A, B, **kwargs):
```

Comment:
Since dissimilarity calculation is a major time-consuming component of partial_align, I've moved this to a separate function so it can be tested independently for all algorithms, and we need only test the partial_align function for a single algorithm (with mocked values for this function when its encountered)